src/docs/sphinx/buildGuide/Prerequisites.rst

```diff
@@ -12,7 +12,7 @@ Minimal requirements:
 
 - `CMake <https://cmake.org/>`_ build system generator (3.17+).
 - build tools (`GNU make <https://www.gnu.org/software/make/>`_ or `ninja <https://ninja-build.org/>`_ on Linux, XCode on MacOS).
-- a C++ compiler with full c++14 standard support (`gcc <https://gcc.gnu.org/>`_ 8.3+ or `clang <https://clang.llvm.org/>`_ 8.0+ are recommended).
+- a C++ compiler with full c++17 standard support (`gcc <https://gcc.gnu.org/>`_ 8.3+ or `clang <https://clang.llvm.org/>`_ 8.0+ are recommended).
```

Comment:
our commonly used versions are now like `gcc12+` and `clang10+`. Shall we update this?

Reply:
I'd say go for it since reflecting what we're actually using/testing on regularly makes the most sense to me.